src/components/InputWithLabel.js

```diff
@@ -21,4 +22,8 @@ const InputWithLabel = ({ children, value, onChange }) => {
   );
 };
 
+InputWithLabel.propTypes = {
+  inputRef: PropTypes.func
```

Comment:
inputRed is a ref, not a prop. Instead of adding it as a type, think about the props that this component accepts and passes down
```suggestion
  children: PropTypes.node.isRequired,
  value: PropTypes.string.isRequired,
  onChange: PropTypes.func.isRequired,
```